modules/ngx_http_cbor_ic_modules/ngx_http_cbor_req_ic_module.c

```diff
@@ -232,8 +160,8 @@ process_request(ngx_http_request_t *r)
     if (ctx == NULL)
         return;
 
-    if (process_body(b, ctx) != PROCESS_OK)
-        return;
```

Comment:
I like this check just because it's a sane default, i.e, even though we don't do anything special with failures, we at least don't pretend they can't happen.

Reply:
Well, it depends on convention "what is an error".

In our case we don't really error at all: if we don't find a variable or can't parse it we just leave it as empty. Adding unused errors might mislead the reader I think. My guess is that if we decide to act on these cases somehow -> then we should re-introduce the handling, proper logging etc.

Reply:
Added back the results